src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,11 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        userService = new UserService();
```

Comment:
init userService once on line 6 - no reason to do it each time when login method is called